partiql-value/src/value.rs

```diff
@@ -15,7 +15,10 @@ mod iter;
 mod logic;
 mod math;
 
-use crate::datum::{Datum, DatumLower, DatumLowerResult, DatumValue};
+use crate::datum::{
+    Datum, DatumCategory, DatumCategoryOwned, DatumCategoryRef, DatumLower, DatumLowerResult,
+    DatumSeqRef, DatumTupleOwned, DatumTupleRef, DatumValue,
```

Comment:
some clippy warnings but could be fixed in a later PR to prevent rebase/merge conflicts. we should probably also investigate at some point why the CI isn't reporting these